crowbar_framework/config/routes.rb

```diff
@@ -275,6 +277,8 @@
       post :prepare
       post :services
       get :services
+      post :controllers
+      get :controllers
```

Comment:
what is the GET route supposed to return? or in other words do we really need it?